int32 y_angle
int32 x_angle
---
sensor_msgs/PointCloud2 point_cloud
int16 exit_code
string error_message
